kustomize/preprod/simple-app/ip.yaml

```diff
@@ -0,0 +1,16 @@
+# Image Policy
+---
+apiVersion: image.toolkit.fluxcd.io/v1beta2
+kind: ImagePolicy
+metadata:
+  name: simple-app-policy
+  namespace: simple-app
+spec:
+  imageRepositoryRef:
+    name: simple-app
+  policy:
+    numerical:
+      order: asc
+  filterTags:
+    pattern: "^RELEASE.(?P<timestamp>.*)Z$"
```

Comment:
In order to have a sortable tag strategy and to not depend on a versioning system (v1.0, v1.0.0, ...), the team would like the tag to contain the branch name, the reverse commit hash and the date

- The pattern of the tag is <branch>-<timestamp>-<reverse commit hash>

Example of tag: preprod-202401020329-831056884

Where 831056884 is the reverse hash of the corresponding commit in simple-app

202401020329 means 2nd January 2024 - 3:29 PM, so in the simple-app repo, we should have the commit that start with 831056884 committed on 2nd January 2024 - 3:29 PM for example.

You will need to change:

- The image policy pattern that extract the timestamp (Check Flux CD doc to understand the regex to use)
-  For this subfolder the branch would be preprod
- on simple-app create a branch preprod and adapt the CI to build the container on preprod branch with the new pattern
- Add a comment in the Deployment to indicate to flux-bot where to edit the tag


Doc: https://fluxcd.io/flux/guides/sortable-image-tags/